mb_relationship_shortcuts.user.js

```diff
@@ -117,11 +116,24 @@ $(document).ready(function(){
             $(this).find("relation-list[target-type='url'] relation").each(function() {
                 var reltype = $(this).attr("type");
                 var target = $(this).children("target").text();
-                if (relationsIconsURLs['url'].hasOwnProperty(reltype)) {
+                if (relationsIconsURLs.url.hasOwnProperty(reltype)) {
+                    $("#" + mbid + " td.relationships").append(
+                        "<a href='" + target.replace(/'/g,"&apos;") + "'>" + "<img style='max-height: 16px;' src='" + relationsIconsURLs.url[reltype] + "' />&nbsp;" + "</a>"
+                    );
+                }
+                if (target.indexOf("d-nb.info") != -1 ) {
```

Comment:
Please declare a global dictionary in the start of the script, like relationsIconsURLs, with all additional sites and their icons.
It's better to separate data and logic processing. This will also avoid duplicating the same code block for each additional site.

Reply:
I will have a look if my skills allow for this. I can't promise anything, though.